### ISSUE TITLE
Can't run setup script

### ISSUE THREAD

**Issue by PakhomovAlexander** [hkdb/VBoxMacSetup#2]
Hello! I can't run setup.sh due to the next explanation: 

> Something is missing... Type "./setup -h" without the quotes to find out more...

So, the command line is: 
`./setup.sh -v MacOS -r 1920x1080`

P.S I've copied this from './setup.sh -h' output.

**Comment by rodman10** [comment]
just change the setup.sh

```bash
 if [ "$#" -le 4 ] && [ "$1" != "-h" ]; then
```

to

```bash
 if [ "$#" -lt 4 ] && [ "$1" != "-h" ]; then
```

**Comment by PakhomovAlexander** [comment]
@rodman10, Thank you! 

I've already done the same thing

**Comment by hkdb** [comment]
Thanks again @PakhomovAlexander !

Your PR has been merged and after adding a couple other things, I have released v1.1

**Comment by hkdb** [comment]
and sorry for the delay all! Work has been hectic!
